### PR TITLE
Your license for the Nanotrasen Robot Voice Module has expired. Please activate your license. [april fools]

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

The voice system that the station's robots use's license has expired, and now defaults to the Free Version voice. Please activate your license to fix it.

## Why It's Good For The Game

This provides an incentive to activate your licenses on software IRL, thereby stimulating the economy and helping with the severe GDP issues we as a country are going through due to the pandemic.

## Changelog
:cl:
tweak: Your license for the Nanotrasen Robot Voice Module has expired. Please activate your license.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
travis please
https://file.house/hArx.png

**Comment by TheSilverNuke** [comment]
Nah we don't need to renew our license. This is perfect.

**Comment by Iamgoofball** [comment]
@TheSilverNuke please explain your reasoning

**Comment by zxaber** [comment]
Now _this_ is an april fools PR.

File: code/modules/mob/living/simple_animal/bot/SuperBeepsky.dm [hunk 1]
```diff
@@ -116,7 +116,7 @@
 			target = C
 			oldtarget_name = C.name
 			speak("Level [threatlevel] infraction alert!")
-			playsound(src, pick('sound/voice/beepsky/criminal.ogg', 'sound/voice/beepsky/justice.ogg', 'sound/voice/beepsky/freeze.ogg'), 50, FALSE)
+			playsound(src, pick('sound/voice/beepsky/criminal.ogg', 'sound/voice/beepsky/justice.ogg', 'sound/voice/beepsky/freeze.ogg', , 'sound/voice/beepsky/activate.ogg'), 50, FALSE)
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/bot/SuperBeepsky.dm]: uh, it looks like there's an extra comma here

**Comment by Jordie0608** [comment]
Which primary schooler did you kidnap to records these files.

Is the 1980s fmv audio quality an intentional part of the joke?

**Comment by Iamgoofball** [comment]
no children were harmed in the making of this PR

**Comment by Iamgoofball** [comment]
also yeah, the low audio quality is part of the joke

**Comment by TheSilverNuke** [comment]
Sad

**Comment by Iamgoofball** [comment]
it'll be testmerged for april fools, we're just doing tracker cleanup

**Pull request closed without merging**